mysql/src/test/java/org/jdbi/v3/mysql/TestMySQL.java

```diff
@@ -104,4 +104,27 @@ public String etag() {
             return etag;
         }
     }
+
+
+    @Test
+    public void testIssue2535PassesSingleLine() {
+        String sqlScript = "CREATE PROCEDURE QWE() "
```

Comment:
Can we use multiline string constants yet?
```
"""
CREATE PROCEDURE 
...
"""
```

Reply:
nope.